### ISSUE TITLE
bug(cdk/menu): With NVDA enabled, keyboard focus doesn't show up 

### ISSUE THREAD

**Issue by zizifn** [angular/components#26080]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

keyboard focus is not working on CDK menu. But seems cdkMenuBar is working fine..

### Reproduction

Steps to reproduce:
1. https://material.angular.io/cdk/menu/examples#cdk-menu-standalone-menu
2. check below GIF for the issue
![a11y-issue](https://user-images.githubusercontent.com/1803942/203821589-228d1f8a-b02e-4946-9839-e6cea9dee673.gif)

### Expected Behavior

With NVDA enabled, keyboard focus can't be focused on first item. And arrow key should be working fine

### Actual Behavior

With NVDA enabled, keyboard focus doesn't focus on first item.  And arrow key not working fine

### Environment

- Angular: 15
- CDK/Material: 15
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by zizifn** [comment]
@crisbeto If you are the right person for this, could you help me check on this? Actually, I did checked the DOM element,  all the aria role seems right in the trigger button and menu. 

I don't know why? Currently, I use `cdkMenuBar` has only one `cdkMenuItem ` as a workaround.  But from my team member testing, seems `cdkMenuBar` has the same issue in macOS VoiceOver. But standalone menu working fine on VoiceOver..
